.github/workflows/static-analysis.yml

```diff
@@ -58,12 +63,21 @@ jobs:
 
       - name: Run PHPStan
         run: ./vendor/bin/phpstan analyse --no-interaction --no-progress --ansi --error-format=sarif > phpstan.sarif
+        continue-on-error: true
```

Comment:
If there's an error, the file won't be generated and subsequent steps will crash, isn't it?

Reply:
It depends. IIRC, if PHPStan detects errors it will exit with a non-zero exit code but still write the file. Default behaviour in GHA would be to stop the job, but we want to at least continue to the next step to upload a SARIF file if it exists. If PHPStan crashed and didn't produce a file, I'm fine with the upload-sarif step failing as well.